helix-term/src/commands/typed.rs

```diff
@@ -2086,6 +2086,16 @@ fn sort_impl(
 
     let selection = doc.selection(view.id);
 
+    if selection.len() == 1 {
+        let (start, end) = selection.line_ranges(text).nth(0).unwrap();
+        let single_selection_spans_multiple_lines = start != end;
+        if single_selection_spans_multiple_lines {
```

Comment:
I don’t think this condition is necessary. The warning is equally relevant if there is a single selection that does not span multiple lines.